app/public/js/update.js

```diff
@@ -22,8 +22,26 @@ setInterval(() => {
     }*/
     if (localSprites.length === 0 && isKeyDown('F'.charCodeAt(0))) {
         //addLocalFallingSpikesSprite();
-        addHomingFireballSprite(350, 700, mainCharacter);
+//        addHomingFireballSprite(350, 700, mainCharacter);
+        addTriggerZone(0, 800, 64, 200, mainCharacter, 1, 250, -150, 0, 0, 150);
     }
+        if (localSprites.length === 0 && isKeyDown('E'.charCodeAt(0))) {
+        //addLocalFallingSpikesSprite();
+//        addHomingFireballSprite(350, 700, mainCharacter);
+        addTriggerZone(0, 800, 64, 200, mainCharacter, 0, 0, 0, 10, -10);
+    }
+    //updating homing fireballs, specifically. Seems like all this code shouldn't be in this file. But translplanting it directly into "updateLocalSprite" didn't work.
+    for (var i = 0; i < localSprites.length; i++) {
```

Comment:
So probably what we want to do eventually is give each `sprite` its own `update` function, and then we just call that here, so this code and the loop below will all just be:
`for (var sprite of localSprites) sprite.update(sprite);`
And we just define a new update function every time we need different behavior, but we don't have to keep editing this code each time.
We would probably do the same thing in `render` if we need to support distinct rendering functions for different sprites.